src/gui/KM_InterfaceGamePlay.pas

```diff
@@ -2822,6 +2850,19 @@ procedure TKMGamePlayInterface.MouseDown(Button: TMouseButton; Shift: TShiftStat
 // 2. Perform SelectingTroopDirection if it is active
 // 3. Display various cursors depending on whats below (might be called often)
 procedure TKMGamePlayInterface.MouseMove(Shift: TShiftState; X,Y: Integer);
+  procedure HandleFieldLMBDrag(P: TKMPoint; aFieldType: TFieldType);
+  begin
+    if not KMSamePoint(LastDragPoint, P) then
+      if (gMySpectator.Hand.CanAddFakeFieldPlan(P, aFieldType)) and (gGameCursor.Tag1 = Byte(cfmPlan)) then

```

Comment:
This is largely my mistake in the past. `Byte(something)` should be really `Ord(something)` throughout the code.